dist/browserscopekeys.yml

```diff
@@ -1,17 +1,17 @@
 ---
-  version: 1
+- version: 1
```

Comment:
The order of these items should be reversed or a change to `lib/browserscope.js` be made to use the last object.

I think the comment at the end of the file should be updated or removed. I don't think it is now or previously was helpful. The comment is perhaps vague or misleading about how the keys were overwriting earlier values in the yaml document.

Since this changes the files schema, another review comment documents a change I think think should be made to the Gruntfile.

Reply:
```diff
diff --git a/lib/browserscope.js b/lib/browserscope.js
index 7d7fcea..f815acd 100644
--- a/lib/browserscope.js
+++ b/lib/browserscope.js
@@ -3,7 +3,7 @@
   var beacon = 0,
       completed = 0,
       failed = false,
-      keys = Rng.Cache.get("browserscopekeys").first(),
+      keys = Rng.Cache.get("browserscopekeys").array[Rng.Cache.get("browserscopekeys").array.length - 1],
       scripts = {
         all: document.createElement("script"),
         rings: document.createElement("script")
```

Reversing is probably more desirable than a change like this.